index.html

```diff
@@ -1,58 +1,161 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
 
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" href="style.css" />
+    <title>Yousef Hesham</title>
+  </head>
 
-  <title>Yousef Hesham</title>
-</head>
-
-<body>
-  <div class="wrapper">
-
-    <header class="head-toolbar">
-      <a class="toolbar-title" href="#">
-        Welcome
-      </a>
-      <nav class="navbar">
-      
+  <body>
+    <div class="wrapper">
+      <header class="head-toolbar">
+        <a class="toolbar-title" href="#"> Welcome </a>
+        <nav class="navbar">
           <a href="#"> <img src="assets/images/hamburger.svg" alt="" /> </a>
-        
-      </nav>
-    </header>
+        </nav>
+      </header>
+
+      <section class="headline">
+        <h2 class="welcome-message">
+          Hey there.I’m Yousef <br />
+          <span>I’m a software developer </span>
+        </h2>
+        <p class="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="social">
+          <ul class="social-icons">
+            <li class="items">
+              <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+            </li>
+            <li class="items">
+              <a href="#">
+                <img src="assets/images/Linkedinicon.svg" alt="" />
+              </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/medium.svg" alt="" /> </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/github.svg" alt="" /> </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/angellist.svg" alt="" /> </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="recent-works">
```

Comment:
Please ensure that the container of the projects has a `display:grid` according to the project requirement.

Reply:
I really forgot that , thank you